app/controllers/api/base_controller.rb

```diff
@@ -22,6 +22,14 @@ class BaseController < ActionController::Base
       render_error 'standard_error', :status => :internal_server_error, :locals => { :exception => error }
     end
 
+    rescue_from NoMethodError do |error|
+      Foreman::Logging.exception("Action failed", error)
+      message = _("Internal Server Error: the server was unable to finish the request. ")
+      message << _("This might be caused by unavailability of some required service, incorrect API call or a bug on server side. ")
+      message << _("Please find more information in server's production.log.")
```

Comment:
`production.log` isn't correct in any other environment. Also, this sounds like an instruction (i.e. "go do this") - I presume we actually want to be passive. Perhaps "There may be more information in the server's logs"?